oak_functions_service/proto/oak_functions.proto

```diff
@@ -63,35 +84,36 @@ message LookupDataChunk {
   repeated LookupDataEntry items = 1;
 }
 
-// Indicates to the LookupDataManager which action should be taken.
-enum UpdateAction {
-  // The sender extends the lookup data builder with this chunk and will send a
-  // further chunk.
-  EXTEND = 0;
-  // The final chunk the sender sends to build the lookup data.
-  FINISH = 1;
-  // The sender needs to abort the building the lookup data.
-  ABORT = 2;
+// If the definition of ExtendNextLookupData changes, the estimation of the size when
+// serialized in the Oak Functions Launcher needs to change, too.
+message ExtendNextLookupDataRequest {
+  LookupDataChunk chunk = 1;
 }
 
-// A message to the LookupDataManager to process the lookup data chunk based on the current
-// action.
-//
-// If the definition of UpdateLookupDataRequest changes, the estimation of the size when
+message ExtendNextLookupDataResponse {
+  bool ok = 1;
+}
```

Comment:
I actually don't remember the details of microRPC (ironically), but isn't the response wrapped in a status / error already? In which case, an error would appear at that level only, and here we can use Empty responses for most methods.